sale_payment_sheet/tests/test_sale_payment_sheet.py

```diff
@@ -76,7 +91,7 @@ def _create_payment_sheet(self):
                 with sheet_form.line_ids.new() as line_sheet:
                     line_sheet.partner_id = self.partner
                     line_sheet.invoice_id = invoice
-                    line_sheet.ref = "REF{}".format(line_sheet.id)
+                    # line_sheet.ref = "REF{}".format(line_sheet.id)
```

Comment:
Why this line is commented?

Reply:
Ref in view is not accesible, with group="base.group_no_one", in this version we can't get this field, for other fields I add full accounting features to the user, but I don't know why for this field and group thid doesn't work